### ISSUE TITLE
Keyboard navigation doesn't work for Find: Project

### ISSUE THREAD

**Issue by rich88** [mgsloan/todoist-shortcuts#159]
Steps to reproduct on Todoist Version 1045 (Chrome, Mac) with Todoist Shortcuts v115

- Go to Inbox
- click in Find box: type name of existing Project
- click Down Arrow + Enter to select the Project name
INCORRECT BEHAVIOR: it edits the highlighted task in the Inbox
EXPECTED BEHAVIOR: should open the Project

Oddly, if you use your mouse to select the project, it DOES open the project.

**Comment by BrunoGomesCoelho** [comment]
Having the exact same problem on Todoist Version 9318 (beta) (Firefox, Linux Mint), Todoist Shortcuts v115.

**Comment by garrulinae** [comment]
Having the same problem on Todoist v1049, Todoist Shortcuts v115, Brave browser, Windows 10.

Note that it's not only relevant to projects, it is applicable to any search item (tasks, filters etc)

When the user hits the Enter key, the selected item in the main pane (task list) receives the focus, instead of the selected item in the search dropdown list.

**Comment by mgsloan** [comment]
Thanks for reporting! Fixed in version 116
